util/secscan/api.py

```diff
@@ -74,27 +74,20 @@ def __init__(self, response):
 _API_METHOD_INSERT = "layers"
 _API_METHOD_GET_LAYER = "layers/%s"
 _API_METHOD_DELETE_LAYER = "layers/%s"
```

Comment:
Remove the delete layer and all the deletion code as well. We should stop calling delete in Clair V2.

Reply:
Removed the `register_model_cleanup_callbacks` from the security scanner interface.

Reply:
Please remove the delete layer call as well

Reply:
Removed all code from `ImplementedSecurityScannerAPI` that performs any sort of write operation on the API.